cttportfolio.css

```
@@ -3,168 +3,481 @@
+.pf_sub_link_div > img{
 	object-fit:cover;
+	height:100%;
+	max-width:100%;
 }
-a.portfoliolinks:hover{
-	border:1px solid black;	
+.pf_sub h6{
+	text-decoration:none;
+	font-weight:600;
+	font-size:2.0rem;
+	padding:5%;
 }
-
 /*======================================== CONTACT SECTION ========================================*/
 .contactme{
-	background-color:red;
+	display:flex;
+	align-items:center;
+	flex-direction:column;
+	margin-top:100px;
 }
 .contactmetext{
 	display:flex;
+	flex-direction:column;
 	justify-content:space-between;
-	padding:0 10%;
+	align-items:center;
+	width:100%;
+}
+.contactmetext > div{
+	margin:50px 0px;
+	padding:50px 50px;
+}
+.contact_div{
+	display:flex;
+	justify-content:space-between;
+	align-items:center;
+	width:80%;
+}
+.contact_div > *{
+	margin:0;
+}
+.contact_div h6{
+	font-size:2rem;
+}
+.contact_input{
+	width:50%;
+}
+.contact_input {
+	margin-top:50px;
+	display:flex;
+}
+.contact_input input{
+	margin-right:50px;
+	width:100%;
+	font-size:2rem;
+	padding-left:10px;
+	border-radius:10px;
+}
+.contact_input button{
+	box-shadow: 0px -2px 5px rgba(248,248,255, 0.0)
+}
+.contact_input button:hover{
+	border:1px solid rgba(248,248,255,0.6);
+	box-shadow: 0px -2px 5px rgba(248,248,255, 0.6)
+}
+.contact_div > h6{
+	font-size:2.0rem;
+}
+.contact_div > p{
+	font-size:2.5rem;
 }
 /*======================================== FOOTER SECTION ========================================*/
 footer{
-	background-color:#ffff66;
 	display:flex;
-	justify-content:space-between;
+	justify-content:center;
+	align-items:center;
 	padding:0 30%;
+	height:300px;
 }
-/*======================================== PROJECT PAGE SECTION ========================================*/
-.projectpageinfo{
+.footer_links{
+	max-width:100%;
+	margin:0 auto;
 	display:flex;
 	justify-content:space-between;
+	align-items:center;
+	height:30%;
+	width:60%;
+}
+.footer_icon_size{
+	width:33%;
+	height:100%;
+}
+.footer_icon_size > a > img{
+	display:block;
+	max-width:100%;
+	max-height:100%;
+	border-radius:50%;
+}
+/*======================================== PROJECT PAGE SECTION ========================================*/
+.pp_body h1, .pp_body h4{
+	font-size:4.0rem;
+}
+.pp_offset{
+	position:relative;
+	top:150px;
+}
+.pp_main{
+	margin:0 auto;
+	width:1300px;
+}
+#background_img{
+	background-image:url("https://i.imgur.com/fsCfb5b.jpg");
+	height:100vh;
+	width:100vw;
+	background-size:cover;
+	position:fixed;
+	z-index:-1;
+}
+.pp_info{
+	display:flex;
+	flex-direction:column;
+	align-items:center;
 	width:1350px;
 	margin:0 auto;
-	height:calc(80vh - 50px);
+	max-height:700px;
 }
-.projectpageinfo > *{
-	width:30%;
+.pp_info_content{
+	display:flex;
+	justify-content:space-between;
+}
+.pp_info_content > *{
+	width:36%;
 }
-.projectpageinfo > img{
+.pp_info_content > img{
 	object-fit:cover;
 }
-.projectpagetext{
+.pp_text{
 	display:flex;
 	flex-direction:column;
 	align-items:center;
 }
-.projectpagelinks{
+.pp_text > p{
+	font-size:2.0rem;
```

Don't need `.0` here.

```suggestion
	font-size:2rem;
```

---

yeah i know but i added this to all I just prefer it with the decimals because still kinda thinkking in pixels. 20px = 2.0 rem instead of 2.